### ISSUE TITLE
Playbook install fails with Ansible 2.9.6 Ubuntu 20.04

### ISSUE THREAD

**Issue by fgjohnson** [Alfresco/alfresco-ansible-deployment#198]
Ansible 2.9.6 in the Ansible deployment fails in Ubuntu 20.04
Indication is it might self resolve with a newer version.

https://github.com/DataDog/ansible-datadog/issues/274

$ ansible -m service_facts localhost

   fatal: [activemq_1]: FAILED! => {"changed": false, "msg": "Malformed output discovered from systemd list-unit-files: accounts-daemon.service                enabled         enabled      "}

$ ansible --version

 ansible 2.9.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/fgjohnson/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.10 (default, Jun  2 2021, 10:49:15) [GCC 9.4.0]

**Comment by alxgomz** [comment]
Sorry but I don't understand how this is related to this project as we are not asking for 2.9.6 specificly and the README page mentions al tests are done with 2.9.18+.
Is it a mistake or maybe can you clarify?

**Comment by fgjohnson** [comment]
Thanks, and sorry...
Figuring out how everything fits together.

I installed/updated a brand new **Ubuntu 20.04**
Installed Ansible with "apt-get install ansible"... and the version I got was 2.9.6... typical newbie mistake!
Downloaded and extracted alfresco-ansible-deployment-master.zip
Ran the playbook for a local install.
This error was thrown and the install didn't complete.
`fatal: [activemq_1]: FAILED! => {"changed": false, "msg": "Malformed output discovered from systemd list-unit-files: accounts-daemon.service enabled enabled "}`

Is Ansible 2.9.18 the recommended version

Installed Ansible by pip3: 2.10.2 the playbook completed successfully. 
Although there are some issues with the Community install that I am experiencing... will try and confirm they aren't already known or something that I can fix.
